bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -41,11 +43,13 @@ object BagsManager {
     def newContext(ctxOpt: Option[ObsContext]): BagsTask =
       BagsTask(observation, curCtx, ctxOpt)
 
-    def performLookup(): Unit = {
+    def performLookup(bagsThread: BagsThread): Unit = {
+      val bagsIdMsg = s"BAGS lookup on thread=${bagsThread.id} for observation=${observation.getObservationID}"
```

Comment:
It seems safer to just use `Thread.currentThread` here because there is really nothing guaranteeing that it's running on `bagsThread`.